tools/ticket-generator/src/main/java/ca/bc/gov/court/traffic/ticket/util/RandomUtil.java

```diff
@@ -651,11 +689,17 @@ public static BaseViolationTicket randomTicket(ViolationTicketVersion version, I
 			Statute count3 = randomStatute();
 			violationTicket.setCount3Description(count3.getDescription());
 			violationTicket.setCount3ActReg(count3.getAct());
-			if (randomBool()) {
+			if (allFilled) {
 				violationTicket.setCount3IsACT("x");
+				violationTicket.setCount3IsREGS("x");
```

Comment:
Is it a valid case for a ticket count to be marked as both act and reg despite the goal being filling up all fields?

Reply:
No, it's not, but that's a business rule and actually not relavent to scanning an image to recognize text. I'm going with the strategy this time to fill out all fields so Form Recognizer knows where fields are located on the image. Later, after the image has been scanned and all text pulled out, we can apply business rules and perform validation on whether some fields should have been populated or not.

Reply:
Oh ok, nice. That makes sense.